src/libs/actions/IOU.js

```diff
@@ -1035,10 +1008,11 @@ export {
     splitBill,
     splitBillAndOpenReport,
     requestMoney,
-    payIOUReport,
     sendMoneyElsewhere,
     sendMoneyViaPaypal,
     payMoneyRequestElsewhere,
     payMoneyRequestViaPaypal,
     setIOUSelectedCurrency,
+    sendMoneyWithWallet,
```

Comment:
NAB but I think it would be more consistent if we move `sendMoneyWithWallet ` below the send money functions and likewise for `payMoneyRequestWithWallet `.